internal/ethapi/api.go

```diff
@@ -1113,12 +1117,19 @@ func (s *PublicTransactionPoolAPI) SendRawTransaction(ctx context.Context, encod
 // Sign calculates an ECDSA signature for:
 // keccack256("\x19Ethereum Signed Message:\n" + len(message) + message).
 //
+// Note, the produced signature conforms to the Ethereum yellow paper spec where
+// the V parameter must be 27 or 28 irrelevant of any ChainID features.
+//
```

Comment:
Instead of using this sentence ("conforms to the Ethereum yellow paper spec...") please document that the format of the resulting signature is `r, s, v where v is 0 or 1`. There is no need to mention ChainID here.